generators/app/templates/service/package.json

```diff
@@ -37,7 +37,9 @@
   },
   "homepage": "https://github.com/<%= orgName %>/<%= projectName %>#readme",
   "dependencies": {
+    <% if (services.indexOf('commerce') > -1) { %>
```

Comment:
I think you can remove this section altogether and use the suggestion I made in the other PR.